sparse_ho/ho.py

```diff
@@ -86,67 +84,67 @@ def _grad_search(
             old_tol = seq_tol[i - 1]
         except Exception:
             old_tol = seq_tol[0]
-        g_func, grad_lambda = _get_val_grad(lambdak, tol=tol, monitor=monitor)
+        val_outer, grad_outer = _get_val_grad(log_alphak, tol=tol, monitor=monitor)
 
-        old_grads.append(norm(grad_lambda))
+        old_grads.append(norm(grad_outer))
```

Comment:
if it holds norm of gradients I would call it grad_norms, old grads make me think it holds full gradients